apps/mull-ui/src/app/pages/home/discover/discover-page.tsx

```diff
@@ -9,7 +9,14 @@ export const DiscoverPage = ({ history }) => {
 
   if (data) {
     const events = (data.discoverEvents as unknown) as Partial<ISerializedEvent>[];
-    var eventCards = events.map((event, index) => (
+    const sortedEvents = events
```

Comment:
Since this sorting function is used in three different places, you can add it to `utilities.ts` to avoid code duplication.